events/2021-kubevirt-summit/proposals/proposal-template.md

```diff
@@ -0,0 +1,28 @@
+# Title
+
+Clear and concise name for the session (not more than 75 chars)
+
+# Abstract
+
+Explain what the session will be about, what will be covered, what will be the
+benefits of attending it / what can attendees expect to get from it).
+
+# Presenters
+
+- Name, title, company, contact details (email, twitter, github - pick any/all)
+
+[ ] The presenters agree to abide by the KubeVirt Code of Conduct
```

Comment:
maybe link? https://github.com/kubevirt/community/blob/master/code-of-conduct.md

Reply:
For events, the CNCF uses the Linux Foundation code of conduct for events, which contains specific information around incidents:
https://events.linuxfoundation.org/about/code-of-conduct/
I believe we should link here instead

Reply:
yup, looks good to me